app/views/rails_admin/main/export.html.erb

```diff
@@ -1,6 +1,6 @@
 <% params = request.params.except(:action, :controller, :utf8, :page, :per_page, :format, :authenticity_token) %>
 <% visible_fields = @model_config.export.with(view: self, object: @abstract_model.model.new, controller: self.controller).visible_fields %>
-<%= form_tag export_path(params.merge(all: true)), method: 'post', class: 'form-horizontal' do %>
```

Comment:
Form elements are now stacked, per [the docs](https://getbootstrap.com/docs/5.0/forms/layout/#horizontal-form) adding the `row` class to form groups should resolve it (dropdown width and cancel button changes appear to be needed as well).

<details>
<summary>Differences</summary>

![Screen Shot 2022-01-07 at 13 25 20](https://user-images.githubusercontent.com/4504083/148609447-7870dfb7-6601-41d9-815d-61b390bc4ced.png)

![Screen Shot 2022-01-07 at 13 26 31](https://user-images.githubusercontent.com/4504083/148609562-d3cf8953-33a1-4364-b354-dc9605dcb937.png)


</details>